### DISCUSSION TITLE
不管内存分配多少或者不分配都会提示内存不足，但在几个星期前都好好的，甚至玩基岩版也会中途闪退，有没有人知道这是怎么回事？

### DISCUSSION THREAD

**Discussion by ArdressEkiyo** [Meloong-Git/PCL#3263] [提问] [answered]
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/14157127/Log1.txt)

**Comment by allMagicNB** [comment]
~~内存虚标~~
你的内存应该有问题
~~基岩版优化比Java版更好~~

**Comment by burningtnt** [comment, marked as answer]
修改系统内存 在桌面右键此电脑 -- 属性 -- 高级系统设置，然后按图里的顺序改成「自动管理所有驱动器分页文件大小」，改完后点三个「确定」，不要点右上角的叉，重启电脑
![image](https://github.com/Hex-Dragon/PCL2/assets/88144530/c148366b-f985-4767-8eb7-7a6df4121348)

**Reply by allMagicNB** [reply]
> 修改系统内存 在桌面右键此电脑 -- 属性 -- 高级系统设置，然后按图里的顺序改成「自动管理所有驱动器分页文件大小」，改完后点三个「确定」，不要点右上角的叉，重启电脑
> ![image](https://github.com/Hex-Dragon/PCL2/assets/88144530/c148366b-f985-4767-8eb7-7a6df4121348)
> 

步骤记录器吧（

**Reply by hejiehao** [reply]
[ZekerZhayard](https://github.com/ZekerZhayard) 做的图

**Comment by ghost** [comment]
> [Log1.txt](https://github.com/Hex-Dragon/PCL2/files/14157127/Log1.txt)

请检查你的 Pagefile （虚拟内存） 大小
如果你用启动脚本，应该会有类似 DOS：分页文件过小之类的提示

**Comment by ZS2147483647** [comment]
我也这样过,虽然不知道原因但重启电脑可以解决所有问题
